### ISSUE TITLE
11

### ISSUE THREAD

**Issue by Ingvarr71** [h0sch180/nougat_platform_manifest#2]
Hello!
When increasing the level of security patches on CrDroid, an error occurs:

CONFLICT (content): Merge conflict in dist / orig / sqlite3.c
error: failed to apply 135afab commit ... 
Fix a buffer overread that could occur when running fts5 prefix queries inside a transaction.
hint: after resolving conflicts, mark the corrected paths
hint: using “git add <path>” or “git rm <path>”
hint: and commit using git commit
Recorded preimage for 'dist / orig / sqlite3.c'
Recorded preimage for 'dist / sqlite3.c'

  all patches are included now

  bump security patch level to 2020-02-05

  script by seluce (until 2019-08-05)

  script by h0sch180 + seluce (2019-09-05 until now)

**Comment by h0sch180** [comment]
Hi
At the moment it's normal, you can ignore this.
The Sqlite patches are not merged at this time, but I have this added because they are a part of the n-asb-2020-02. you can see it here:
https://review.lineageos.org/q/topic:%22n-asb-2020-02%22+(status:open%20OR%20status:merged)

The lineage 14.1 Team takes this month a failure and merged not all monthly patches to the same time.

**Comment by Ingvarr71** [comment]
I can continue to collect firmware further?

**Comment by h0sch180** [comment]
yes, you can.
It's not so importend.

**Comment by Ingvarr71** [comment]
On CrDroid, applying security patches takes a very long time, it took me about 6 hours, although on RR it took about 15 minutes.

**Comment by h0sch180** [comment]
yes, I know. I have the same "problem", but at the moment I don't know why.
But it takes only the first time so long.
Next month you can go this way:
`repo forall -vc "git reset --hard"`
`repo sync --force-sync`

then go to the n_asb/crdroid folder and start the "apply-patches.sh"

With this workaround you need only 1 Minute to apply the patches.
You can do the same with RR and ViperOS.

**Comment by Ingvarr71** [comment]
Thanks for the help.
And thanks for supporting the nougat firmware

**Comment by h0sch180** [comment]
np, my boys have the Cubot Max and Oukitel K6000Pro, so I must support nougat. :)
Seluce stop the nougat support so I do it alone, it's not so difficult and need not a lot of time.
When you have problems, don't hesitate to contact me.

**Comment by Ingvarr71** [comment]
I collect for Doogee T6 pro

**Comment by Ingvarr71** [comment]
OK, deal.
